automap/src/comm_layer/pmp.rs

```diff
@@ -1445,8 +1857,13 @@ mod tests {
         let mut last_remapped = Instant::now().sub(Duration::from_secs(3600));
```

Comment:
Doesn't matter--this isn't a timeout. It's kind of the opposite. We're claiming that the last time the port was remapped was an hour ago, which means that it needs to be remapped RIGHT NOW! So instead of waiting until it's time to remap, the test remaps immediately and runs quickly.